nixos/modules/services/security/oauth2_proxy.nix

```diff
@@ -12,7 +12,7 @@ let
   # command-line to launch oauth2_proxy.
   providerSpecificOptions = {
     azure = cfg: {
-      azure.tenant = cfg.azure.tenant;
+      azure-tenant-string = cfg.azure.tenant;
```

Comment:
I don't think this option is called azure-tenant-string. (https://github.com/oauth2-proxy/oauth2-proxy/blob/c7bfbdecef0a724b5bc752607edd4257928ddd40/main.go#L65)

Reply:
yes, you're right, I'll fix that.